docs/source/using_the_ve/configuration/config.md

```diff
@@ -28,14 +28,21 @@ language_info:
 This module is used to configure a `virtual_ecosystem` simulation run. This module
 reads in a set of configuration files written using `toml`. It is setup in such a way as
 to allow a reduced set of modules to be configured (e.g. just `plants` and `soil`), and
-to allow specific module implementations to be configured (e.g. `plants_with_hydro`
-instead of `plants`). The resulting combined model configuration is validated against a
-set of [`JSON Schema`](https://json-schema.org). If this passes, a combined output file is
-saved as a permanent record of the model configuration. This configuration is also saved
-as a dictionary accessible to other modules and scripts.
+to allow specific module implementations to be configured (e.g. `abiotic_simple`
+instead of `abiotic`). It deliberately accepts multuple configuration files in order to
```

Comment:
```suggestion
instead of `abiotic`). It deliberately accepts multiple configuration files in order to
```

Reply:
Btw there is a pre-commit hook for the `codespell` tool that can be useful for typos like this. We use it in the [`python-template` repo](https://github.com/ImperialCollegeLondon/python-template/).